source/includes/steps-repo.yaml

```diff
@@ -133,7 +133,22 @@ content: |
   .. code-block:: sh
 
      git remote add upstream https://github.com/<repoOwner>/<remoteRepoRoot>.git
+---
+title: Remove the redirects file
+ref: remove-redirects
+content: |
+
+  Some of our repositories have a ``redirects`` file that redirects browsers permanently to a new location.
```

Comment:
I'd consider adding something that the only way the redirect is removed is when the browser cache is cleared.

Reply:
We really need to discourage people from modifying this file or leaving it available for others to modify.